server/api/elections.py

```diff
@@ -50,6 +56,7 @@ def create_election():
         id=str(uuid.uuid4()),
         audit_name=election["auditName"],
         audit_type=election["auditType"],
+        ballot_polling_type=election["ballotPollingType"],
```

Comment:
if `ballotPollingType` is not a required field, then if it's not present in the request body, this will fail. you can use `election.get("ballotPollingType")` to defend against that